sorc/ncep_post.fd/AVIATION.f

```diff
@@ -360,42 +373,106 @@ SUBROUTINE CALCAT(U,V,H,U_OLD,V_OLD,H_OLD,CAT)
 !cvg=-(du/dx+dv/dy)
            CVG = -( (U(I+IHE(J),J) - U(I+IHW(J),J))*10000./(2*DX(I,J)) &
                   +(V(I,J+1) - V(I,J-1))*10000./(2*DY(I,J)) )
+           ELSE
+            DEF = SPVAL
```

Comment:
line 377, DEF should be DSH.
@JesseMeng-NOAA

Reply:
DEF and CVG are the final computed variables from this block.